cpp/src/arrow/record_batch_test.cc

```diff
@@ -499,27 +499,27 @@ TEST_F(TestRecordBatchReader, ToTable) {
 ARROW_SUPPRESS_DEPRECATION_WARNING
 TEST_F(TestRecordBatchReader, DeprecatedReadAllToRecordBatches) {
```

Comment:
Sounds like the entire test may not be needed?

Reply:
Ahh you're right. No it doesn't look necessary...